### ISSUE TITLE
#bug # accessibility.

### ISSUE THREAD

**Issue by alireza1385mamani** [UnigramDev/Unigram#2615]
**Describe the bug**
the translate and reactions butten are not accessible for screenreaders.

**To Reproduce**

**Expected behavior**

**Screenshots**

**Version Info**
 - Unigram: [e.g. 7.8.6339 x64]
 - Windows: [e.g. October 2018 update]

**Additional context**

**Comment by FrayxRulez** [comment]
- You must enable Translation from Settings > Languages
- Reactions aren't currently supported
